test/slack/command.test.js

```diff
@@ -39,4 +39,48 @@ describe('Command class', () => {
       'area/api',
     ]);
   });
+
+  describe('normalizes unicode quotes', () => {
+    test("supports '\u00AB'", () => {
+      const command = new Command({
+        text: 'subscribe integration/jira +label:«help wanted«',
```

Comment:
I considered that to base the tests on real world use cases, but then I decided to add tests per specific character instead to avoid a combinatorial explosion and discussions about correct usage.
Thus the intend was to cover the real cases by implication.
However, I can make an exception for this particular case,  but I am not going to cover all (correct and incorrect) combinations 😬